spec/acceptance/clone_repo_spec.rb

```diff
@@ -1,17 +1,18 @@
 require 'spec_helper_acceptance'
 
-tmpdir = default.tmpdir('vcsrepo')
+tmpdir = '/tmp/vcsrepo'
```

Comment:
Don't think this will be an issue with any of the OSs we've defined in the `provision.yml`, but, I wonder if we should consider [Ruby's own tempdir functionality](https://ruby-doc.org/stdlib-2.5.1/libdoc/tmpdir/rdoc/Dir.html)?

Reply:
@cmccrisken-puppet sure making the change

Reply:
thank you

Reply:
More of a question from my perspective - do feel free to dismiss / ignore 😆

Reply:
If you do go down that road, I believe there's nice resource handling. Whether it conflicts with the RSpec lib, I'm not sure (_unlikely_):
```
Dir.mktmpdir do |tmpdir|
   // All the rest of the code
}
```

Reply:
Thank you